### ISSUE TITLE
Add endpoint for exposing the code.json file

### ISSUE THREAD

**Issue by IanLee1521** [GSA/code-gov-api#202]
It would be great to add an endpoint to the REST API to enable an end user to get the full code.json file for all the federal agencies. That would allow comparisons against the data from agencies. 

For instance, I am interesting in comparing Federal wide information to the data at DOECode. Additionally, I have am investigating other integrations with, for instance, GitHub.com/llnl/scraper

**Comment by froi** [comment]
@IanLee1521 this already exists. The endpoint `/status/{agencyName}/fetched` returns the code.json that was fetched in our last harvest.

You can try it out [here](https://api.code.gov/docs/#/status/get_status__agencyName__fetched). You do not need an API key to view this endpoint.

I will admit that the endpoint and the description in the swagger docs aren't descriptive enough to let you know their functionality. I'll keep this issue open and work on making them better.

**Comment by froi** [comment]
I'll add another thing to this. The endpoint signature should say `{agencyAcronym` not `{agencyName}`.

Eg. `curl -X GET https://api.code.gov/status/GSA/fetched`

I've taken note of it and will be fixing this as well.

**Comment by IanLee1521** [comment]
Thanks @froi -- Is there (could there be) a similar endpoint for pulling the / a combined code.json file?

**Comment by froi** [comment]
There could be but it isn't on our roadmap. We used something similar to that for our front-end when it wasn't using the API. You can take a look at an example [here](https://github.com/GSA/code-gov-data/blob/master/releases.json). Keep in mind that this file is 4 months out of date.

The file was created by the [code-gov-harvester](https://github.com/GSA/code-gov-harvester). You could run it yourself and reproduce the file if you think it might be useful to you.

**Comment by IanLee1521** [comment]
Got it. Perhaps that would be a good feature request to add into https://github.com/llnl/scraper to handle creating that...?

As an aside, do you have a static list of the agencyAcronym's that you're using?

**Comment by froi** [comment]
> As an aside, do you have a static list of the agencyAcronym's that you're using?

Yes, [agency_metadata.json](https://github.com/GSA/code-gov-api/blob/master/config/agency_metadata.json). We follow whatever acronym the agencies are using in their `code.json`.
 
__Eg.__
```json
{
    "id": 1,
    "name": "Department of Agriculture",
    "acronym": "USDA",
    "website": "https://usda.gov/",
    "codeUrl": "https://www.usda.gov/code.json",
    "fallback_file": "USDA.json",
    "requirements": {
      "agencyWidePolicy": 0.75,
      "openSourceRequirement": 0,
      "inventoryRequirement": 0,
      "schemaFormat": 0.5,
      "overallCompliance": 0
    },
    "complianceDashboard": true
}
```
